BricksAndHearts.UnitTests/ControllerTests/Landlord/LandlordControllerTests.cs

```diff
@@ -25,6 +25,27 @@ public void RegisterGet_CalledByUnregisteredUser_ReturnsRegisterViewWithEmail()
             .Which.Email.Should().Be(unregisteredUser.GoogleEmail);
     }
 
+    [Fact]
+    public async void RegisterPost_CalledByUnregisteredUser_ReturnsProfile()
+    {
+        // Arrange
+        var unregisteredUser = CreateUnregisteredUser();
+        MakeUserPrincipalInController(unregisteredUser, UnderTest);
+        var formResultModel = new LandlordProfileModel();
+        var returnedLandlord = new LandlordDbModel();
+
+        A.CallTo(() => LandlordService.RegisterLandlord(formResultModel, unregisteredUser))
+            .Returns((ILandlordService.LandlordRegistrationResult.Success, returnedLandlord));
+
+        // Act
+        var result = await UnderTest.RegisterPost(formResultModel) as RedirectToActionResult;
+
+        // Assert
+        result.Should().BeOfType<RedirectToActionResult>();
+        result.Should().NotBeNull();
+        result!.ActionName.Should().BeEquivalentTo("MyProfile");

```

Comment:
Could also assert that the call to the service has happened

Reply:
I was just thinking that as I reviewed Claire's code and saw she had 😅

Reply:
Added the lines in to check there was or wasn't calls to specific services :)